src/Logger/SimpleHttpLogger.php

```diff
@@ -71,7 +71,7 @@ public function trace($spans)
     protected function validResponse($headers)
     {
         foreach ($headers as $header) {
-            if (preg_match('/202/', $header)) {
+            if (preg_match('/^HTTP\/\S+\s+202/', $header)) {
```

Comment:
Can you give an example of one of these response code headers so I can just sanity check that this regex matches it?

Reply:
one that I've seen is `HTTP/1.1 200 OK`

Reply:
the underlying point is valid so I added a change with a comment